CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 ### Added
 - workaround by @semaeostomea to support arabic and right-to-left scripts: [link to documentation](https://pyfpdf.github.io/fpdf2/Unicode.html#arabic-script-workaround)
 - documentation on shapes styling [link to documentation](https://pyfpdf.github.io/fpdf2/Shapes.html#path-styling)
+- new function `set_page_background()` added by @semaeostomea
```

Comment:
Could you maybe insert a link pointing to the method docstring please?
https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.set_page_background